core/src/main/kotlin/model/documentableProperties.kt

```diff
@@ -39,7 +39,7 @@ object ObviousMember : ExtraProperty<Documentable>, ExtraProperty.Key<Documentab
     override val key: ExtraProperty.Key<Documentable, *> = this
 }
 
-data class CheckedExceptions(val exceptions: SourceSetDependent<List<DRI>>) : ExtraProperty<Documentable>, ExtraProperty.Key<Documentable, ObviousMember> {
+data class CheckedExceptions(val exceptions: SourceSetDependent<List<DRI>>) : ExtraProperty<Documentable> {
```

Comment:
It is not related to the change itself. I stumbled on it and thought I can d a little bit of community service and remove it.

It is leftover from copying the code and it is never correct to use `CheckedException` class instance as a property key.
`ExtraProperty.Key<Documentable, ObviousMember>` means that values of this type can be used as keys to extract something of type `ObviousMember` from `Documentable`, which doesn't make sense since `ObviousMember` is an object and defines itself as a key. That means that only object itself can be used as its key in `Documetable`'s property container.

Reply:
Thanks for the explanation :)